tests/py/dynamo/conversion/harness.py

```diff
@@ -29,6 +32,25 @@
 _LOGGER: logging.Logger = logging.getLogger(__name__)
 
 
+# this is to enable dynamo tracer as True in the converter test files batch by batch
+def get_use_dynamo_tracer(use_dynamo_tracer: Any) -> bool:
```

Comment:
why do we need this? any test using our harness should be on dynamo right? the fx tests are in the py/ tree

Reply:
@narendasan  since there are 74 testfiles(~500 testcases) are impacted by switching from fx.symbolic_tracer to dynamo tracer, I use this to method to split the changes batch by batch
so the first batch is to automatically set use_dynamo_tracer to True for the below operators starts with a:
tests/py/dynamo/conversion/test_a*.py
I will eventually migrate all the symbolic tracer to dynamo tracer and remove this temporary helper method.